### PR TITLE
Resolve class aliases when accessing properties

### PR DESCRIPTION
**Pull request by weirdan**
Fixes vimeo/psalm#5056

### CODE REVIEW

File: src/Psalm/Internal/Codebase/Properties.php [hunk 1]
```diff
@@ -305,4 +316,18 @@ public function getPropertyType(
 
         return null;
     }
+
+    /**
+     * Returns ClassLikeStorage for the given class name, resolving aliases
+     */
+    private function getClasslikeStorage(string $fq_class_name): ?ClassLikeStorage
```
Comment (muglug) [inline src/Psalm/Internal/Codebase/Properties.php:323]: Might it make sense to move this into ClassLikeStorageProvider like `getSafe` or such?

Comment (weirdan) [reply src/Psalm/Internal/Codebase/Properties.php:323]: The method uses `Classlikes::getUnAliasedName()` so if we move it the provider would have to depend on `Classlikes` which itself depends on `ClassLikeStorageProvider`. Such circular dependency would complicate initialization of both.

Comment (weirdan) [reply src/Psalm/Internal/Codebase/Properties.php]: Done

**Comment by muglug** [comment]
Thanks!

**Pull request merged**